deposit_contract/tests/contracts/test_deposit.py

```diff
@@ -118,14 +160,28 @@ def test_deposit_event_log(registration_contract, a0, w3, deposit_input):
         assert log['index'] == i.to_bytes(8, 'little')
 
 
-def test_deposit_tree(registration_contract, w3, assert_tx_failed, deposit_input):
+def test_deposit_tree(registration_contract, w3, assert_tx_failed):
     log_filter = registration_contract.events.DepositEvent.createFilter(
         fromBlock='latest',
     )
 
     deposit_amount_list = [randint(MIN_DEPOSIT_AMOUNT, FULL_DEPOSIT_AMOUNT * 2) for _ in range(10)]
     deposit_data_list = []
     for i in range(0, 10):
+        deposit_data = DepositData(
+            pubkey=SAMPLE_PUBKEY,
+            withdrawal_credentials=SAMPLE_WITHDRAWAL_CREDENTIALS,
+            amount=deposit_amount_list[i],
+            signature=SAMPLE_VALID_SIGNATURE,
+        )
+        deposit_input = (
+            SAMPLE_PUBKEY,
+            SAMPLE_WITHDRAWAL_CREDENTIALS,
+            SAMPLE_VALID_SIGNATURE,
+            hash_tree_root(deposit_data),
+        )
+        deposit_data_list.append(deposit_data)
```

Comment:
Need to add `registration_contract.functions.get_deposit_count().call()` at the end of this test to verify that the compiler bug is fixed and _stays_ fixed.

See https://github.com/ethereum/eth2.0-specs/issues/1341

Reply:
Done :)